.github/workflows/_reusable_app_release.yml

```diff
@@ -4,6 +4,10 @@ on:
       fastlane_action:
         required: true
         type: string
+      cloud:
+        required: true
+        type: boolean
+        default: true
```

Comment:
I think the name confused me, that we want to load the changeling to/from the cloud like AWS?
But I think what is meant is "this s a build to use the wire cloud as backend", right?

Maybe we can use a name like `is_cloud_build`?

Reply:
relates to this configuration: https://github.com/wireapp/wire-ios-build-assets/tree/master/Cloud/AppStore

Reply:
done